pkg/META.in

```diff
@@ -53,7 +53,7 @@ package "eq" (
   version = "%{version}%"
   description = "[@@deriving eq]"
   requires(-ppx_driver) = "ppx_deriving"
-  ppxopt(-ppx_driver,-custom_ppx) = "ppx_deriving,./ppx_deriving_eq.cma"
+  ppxopt(-ppx_driver,-custom_ppx) = "ppx_deriving,-deriving-plugin ./ppx_deriving_eq.cma"
```

Comment:
Yes it is failing, there are at least two issues.

First, contrary to what findlib announces, spaces are not accepted. Although it writes `ppxopt variable must include package name, e.g. ppxopt="foo,-name bar"`, literally writing `ppxopt="foo,-name bar"` will fail too.
A solution could be to write `-deriving-plugin=./ppx_deriving_eq.cma`. But it seems the `=` syntax is not parsed by Arg module on 4.02, which is used by `ocaml-migrate-parsetree`.

Second, you are right: when an argument begins with `./`, findlib prepends the current working directory. This breaks with the current patch.
And using multiple `ppxopt` is rejected too.